Cargo.toml

```diff
@@ -70,6 +70,7 @@ ignore = "0.4.20"
 rayon = "1.7.0"
 tree-sitter = "0.20.9"
 similar = { version = "2.2.1", features = ["inline"] }
+serial_test = "2.0.0"
```

Comment:
The `serial_test` is only for development environment, so this should be installed es `dev-dependencies`